.github/workflows/publish_npm.yml

```diff
@@ -0,0 +1,34 @@
+name: 'publish npm package'
+
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    name: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out the repo'
+        uses: 'actions/checkout@v3'
+        with:
+          submodules: 'recursive'
```

Comment:
Not necessary unless we're using git submodules in the repo

Reply:
Sorry I wasn't clear. You do need the `'actions/checkout@v3'` step, you just don't need the with submodules: 'recursive' line.